tensorboard/backend/event_processing/directory_watcher.py

```diff
@@ -167,7 +171,7 @@ def _InitializeLoader(self):
     if path:
       self._SetPath(path)
     else:
-      raise StopIteration
+      return
```

Comment:
Might as well just omit the entire else clause, right?

Reply:
Haha yes of course, done.  Not enough coffee this morning apparently.